style.css

```diff
@@ -0,0 +1,1014 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap');
+
+:root {
+  --primary: #272a31;
+  --secondary: #ec5242;
+  --light: #d3d3d3;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+}
+
+.material-symbols-outlined {
+  font-variation-settings:
+    'FILL' 0,
+    'wght' 800,
+    'GRAD' 0,
+    'opsz' 48;
+}
+
+/* header */
+
+.account {
+  display: none;
+}
+
+header {
+  align-self: flex-end;
+  width: 100%;
+}
+
+nav {
+  width: 100%;
+  padding: 17.5px 5%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  background-color: transparent;
+}
+
+nav img,
+.menu-items {
+  display: none;
+}
+
+nav .material-symbols-outlined {
+  font-size: 40px;
+  color: var(--primary);
+}
+
+/* Popup-nav */
+
+.popup-nav {
+  display: none;
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  background-color: rgba(62, 53, 53, 0.861);
+  backdrop-filter: blur(8px);
+}
+
+.popup-nav-visible {
+  display: flex;
+}
+
+.popup-header {
+  display: flex;
+  justify-content: flex-start;
+}
+
+.popup-menu {
+  display: grid;
+  align-items: center;
+  justify-content: center;
+  margin-top: 5%;
+  width: 100%;
+}
+
```

Comment:
- [ ] Please make sure to align the hamburger menu to the left instead of right to match the [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) layout and positioning as it is mentioned in the [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#project-requirements):

> You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
> - Layout: composition and space between elements.

Behance design:

<img width="393" alt="Screenshot 2023-03-18 at 12 58 35 PM" src="https://user-images.githubusercontent.com/44798044/226093792-15d99ffc-66b7-4422-b7f2-4457a02ee49f.png">

Your Design:

<img width="362" alt="Screenshot 2023-03-18 at 12 42 55 PM" src="https://user-images.githubusercontent.com/44798044/226093801-77d5a8ff-4ca1-4959-a867-451b5a2b2123.png">